docs/csharp/how-to/search-strings.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "How to: search strings (C# Guide)"
+ms.date: 02/21/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "searching strings [C#]"
+  - "strings [C#], searching with String methods"
+  - "strings [C#], searching with regular expressions"
+ms.assetid: fb1d9a6d-598d-4a35-bd5f-b86012edcb2b
+ms.author: "wiwagn"
+---
+
+# How to: search strings
+
+You have two main strategies that you can use to search for text in strings. Member methods of the <xref:System.String> class search for specific text. Regular expressions search for patterns in text.
```

Comment:
The first sentence is repetitive. Better: "You can use two main strategies..."
"Member methods" --> just "Methods"?

Reply:
By the way, I've read "member methods" as "non-static methods".